### PR TITLE
Switch ourselves to main

### PR DESCRIPTION
**Pull request by 72636c**
Fostering a respectful workplace. Closes off #221 and makes Sam's life more difficult in #1497.

### CODE REVIEW

**Comment by 72636c** [comment]
I don't know if I'm meant to merge this into `master` first or to split off `main` at this point 🤷

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -102,6 +102,6 @@ jobs:
       - name: Set up pnpm
         run: corepack enable pnpm
 
-      - if: github.head_ref != 'beta' && github.head_ref != 'changeset-release/master' && github.ref_name != 'beta' && github.ref_name != 'changeset-release/master'
+      - if: github.head_ref != 'changeset-release/master' && github.ref_name != 'changeset-release/master'
```
Comment (samchungy) [inline .github/workflows/validate.yml:105]: Should this be main?

**Comment by samchungy** [comment]
> I don't know if I'm meant to merge this into `master` first or to split off `main` at this point 🤷

Switch the target to main after we can see the comparison?

File: .github/workflows/validate.yml (hunk 1 above)
Comment (72636c) [reply .github/workflows/validate.yml:105]: 💯

```suggestion
      - if: github.head_ref != 'changeset-release/main' && github.ref_name != 'changeset-release/main'
```

**Review by samchungy — APPROVED** [review]

**Comment by 72636c** [comment]
Oh f

**Pull request merged**